examples/gitops-kubevirt/02-auto-import/full-auto/observability/watch-script.sh

```
@@ -0,0 +1,201 @@
+    echo "Check cluster status manually:"
+    echo ""
+    echo "  oc get hostedcluster -n ${NAMESPACE} ${CLUSTER_NAME}"
+    echo "  oc get managedcluster ${CLUSTER_NAME}"
+    echo "  oc logs -n ${NAMESPACE} deployment/hypershift-operator"
+    echo ""
+    exit 1
+  fi
+
+  # Check if deployment is complete
+  if check_hostedcluster &>/dev/null && check_managedcluster &>/dev/null && check_nodepool &>/dev/null; then
+    local hc_available=$(oc get hostedcluster -n "${NAMESPACE}" "${CLUSTER_NAME}" -o jsonpath='{.status.conditions[?(@.type=="Available")].status}' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)